editor/plugins/canvas_item_editor_plugin.cpp

```diff
@@ -4875,14 +4877,25 @@ void CanvasItemEditor::_popup_callback(int p_op) {
 		case SKELETON_MAKE_BONES: {
 			HashMap<Node *, Object *> &selection = editor_selection->get_selection();
 			Node *editor_root = get_tree()->get_edited_scene_root();
+			Vector<Node2D *> selected_nodes;
```

Comment:
```suggestion
			Vector<Node2D *> selected_nodes;
			selected_nodes.reserve(selection.size());
```
Also move this after the condition below.